app/src/main/java/com/yacgroup/yacguide/activity_properties/AscentFilterable.kt

```diff
@@ -18,12 +18,13 @@
 package com.yacgroup.yacguide.activity_properties
 
 import android.content.Intent
+import androidx.viewbinding.ViewBinding
 import com.yacgroup.yacguide.R
 import com.yacgroup.yacguide.RouteActivity
 import com.yacgroup.yacguide.TableActivityWithOptionsMenu
 import com.yacgroup.yacguide.utils.IntentConstants
 
-class AscentFilterable(private val _activity: TableActivityWithOptionsMenu) : ActivityProperty {
+class AscentFilterable<bindingType: ViewBinding>(private val _activity: TableActivityWithOptionsMenu<bindingType>) : ActivityProperty {
```

Comment:
Please you a capital letter for types: `ViewBindingType` instead of `bindingType`